Formula/john-jumbo.rb

```diff
@@ -53,12 +53,25 @@ class JohnJumbo < Formula
     sha256 "65a4aacc22f82004e102607c03149395e81c7b6104715e5b90b4bbc016e5e0f7"
   end
 
+  # Upstream M1/ARM64 Support.
+  # Combined diff of the following four commits, minus the doc changes
+  # that block this formula from using these commits otherwise.
+  # https://github.com/openwall/john/commit/d6c87924b85323b82994ce01724d6e458223fd36
+  # https://github.com/openwall/john/commit/d531f97180a6e5ae52e21db177727a17a76bd2b4
+  # https://github.com/openwall/john/commit/c9825e688d1fb9fdd8942ceb0a6b4457b0f9f9b4
+  # https://github.com/openwall/john/commit/716279addd5a0870620fac8a6e944916b2228cc2
+  patch do
+    url "https://gist.githubusercontent.com/DomT4/80e8e8f5477c91c124b727df2ace55c7/raw/022a60fbae8ac1f913c719da6c7152620beaf511/john_jumbo_m1.diff"
```

Comment:
Can we rehost this in homebrew/Homebrew-patches?

Reply:
I should read other comments first 😅

Reply:
Heh 😄. I'll move the patch over later today & update the PR, yeah.